### ISSUE TITLE
InspecIT don't work with Tomcat 7.0.92

### ISSUE THREAD

**Issue by mikefaille** [inspectIT/inspectIT#416]
I'm using java version "1.8.0_202" on macOSX

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access using Lookup on org.springframework.objenesis.instantiator.util.DefineClassHelper$Java11 (file:/Users/michael/src/apache-tomcat-7.0.94/temp/10301846683721451166.jar) to interface java.sql.Statement
WARNING: Please consider reporting this to the maintainers of org.springframework.objenesis.instantiator.util.DefineClassHelper$Java11
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by mariusoe** [comment]
Hi @mikefaille 
I never saw this warning before, but the message looks like that this error is related to Spring. As far as I know, spring itself uses quite a lot reflection stuff and maybe there are some problems with accessing/interacting classes which have been instrumented by our agent - the `java.sql.Statement` is one of the classes which are instrumetned by default.
It looks like that this is "just" a warning and not really an error.

Nevertheless, this project is currently not actively maintained anymore, due to the fact that we started a successor version of inspectIT: https://github.com/inspectIT/inspectit-ocelot
Its approach is a bit different than the original (this) inspectIT but maybe it is also fitting your needs, so maybe give it a try and test if this problem is fixed there.

So it's likely that we cannot provide a fix for this problem in the near future or at all - at least in this project.

Greets,
Marius

**Comment by mikefaille** [comment]
inspectit-ocelot seems to works !
